recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 build:
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 1
+  number: 0
```

Comment:
At conda forge. This number should go down only if the version goes up. Do you mean to increase the version to??

Reply:
Ah so I can update the version number and checksum in this PR itself, it doesn't have to be in the one from the bot?

Reply:
yeah the bot is there to help.

Since you aren't a maintainer (yet!) you can't edit the bot's branch.

Best to have "all the fixes in 1 branch" and merge that once done!

Reply:
ah ok, I don't do any conda stuff much 

I'm more than happy for that you're maintaining the conda side of my library :smile: